HopsanCore/include/Parameters.h

```diff
@@ -82,6 +85,7 @@ class HOPSANCORE_DLLAPI ParameterEvaluator
     size_t mDepthCounter;
     ParameterEvaluatorHandler* mpParameterEvaluatorHandler;
     std::vector<HString> mConditions;
+    bool mTriggersReconfiguration = false;
```

Comment:
Umm I think this should have failed on non c++03 export, but it did not ?
I dont think initializing variables directly in header should work.